assets/js/metrics_live/index.js

```diff
@@ -10,6 +10,16 @@ const SeriesValue = (options) => {
   }
 }
 
+const XSeriesValue = (options) => {
+  return {
+    value: (self, v) => {
+      if (v === null) return '';
+      let d = new Date(v * 1e3)
+      return uPlot.fmtDate('{YYYY}-{MM}-{DD} {HH}:{mm}:{ss}')(d)
```

Comment:
@LostKobrakai you should pull this formatter out of `value()` so it's not re-parsed and re-created every time.

```js
const fmtr = uPlot.fmtDate('{YYYY}-{MM}-{DD} {HH}:{mm}:{ss}');

value(self, v) {
  return fmtr(d);
}
```

Reply:
or a better route would be the other signature which will do it for you:

```js
value: '{YYYY}-{MM}-{DD} {HH}:{mm}:{ss}'
```

https://github.com/leeoniya/uPlot/issues/83#issuecomment-570416433